app/index.html

```diff
@@ -85,12 +85,16 @@ <h2>Get Started in 1 Minute</h2>
       <div class="one-half column">
         <ol>
           <li>
-            <h4>Add your API on Kong</h4>
-            <p>After <a href="/install">installing</a> and starting Kong, use the Admin API on port 8001 to add a new API. Kong will route every incoming request with the specified public DNS to the associated target URL.<br/><br/></p>
+            <h4>Register a Service in Kong</h4>
+            <p>After <a href="/install">installing</a> and starting Kong, use the Admin API on port 8001 to add a new Service. Services represent upstream servers containing an API.</p>
```

Comment:
suggestion: containing an API -> exposing an API

(Also, I don't know, but "Register" sounds a bit strange to me... I'm not sure if it's because the word has some other overtones (e.g. "register this software"), or because "registering a service" sounds like more work than "adding a service". Anyway, feel free to ignore.)

Reply:
How about "Configure a Service in Kong" then?